core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -2224,6 +2224,13 @@ private static RelDataType deriveTypeMapFromEntries(SqlOperatorBinding opBinding
           OperandTypes.NUMERIC_OPTIONAL_NUMERIC,
           SqlFunctionCategory.NUMERIC);
 
+  /** The "LOG(numeric, numeric1)" function. Returns the base numeric1 logarithm of numeric. */
+  @LibraryOperator(libraries = {MYSQL, SPARK})
+  public static final SqlFunction LOG_MYSQL =
```

Comment:
If both `MYSQL` and `SPARK` are supported, this `LOG_MYSQL` may not be accurate?

Reply:
I wrote log_mysql_spark earlier, but @NobiGo  recommend log_mysql.
I can't think of a good way to name it. Maybe we should agree on a standard

Reply:
> I wrote log_mysql_spark earlier, but @NobiGo recommend log_mysql. I can't think of a good way to name it. Maybe we should agree on a standard

`LOG_BASE`？

Reply:
What do you think? @NobiGo

Reply:
LOG_MYSQL means we use MYSQL mode. same as others name like  SUBSTR_POSTGRESQL. So I think LOG_MYSQL is good.  Don't need to modifty it.

Reply:
> LOG_MYSQL means we use MYSQL mode. same as others name like SUBSTR_POSTGRESQL. So I think LOG_MYSQL is good. Don't need to modifty it.

Here SUBSTR_POSTGRESQL will only use POSTGRESQL MODE, while the method LOG_MYSQL here will be used by both lib modes, there is a difference here.